src/mainAppBar/userProfile.component.test.tsx

```diff
@@ -105,14 +93,24 @@ describe('User profile component', () => {
       avatarUrl: 'test_url',
     };
     const wrapper = mount(
-      <MuiThemeProvider theme={theme}>
-        <UserProfileComponent store={mockStore(state)} />
-      </MuiThemeProvider>
+      <StyledEngineProvider injectFirst>
+        <ThemeProvider theme={theme}>
+          <UserProfileComponent store={mockStore(state)} />
+        </ThemeProvider>
+      </StyledEngineProvider>
     );
 
     expect(wrapper.find('#simple-menu').first().prop('open')).toBeFalsy();
 
-    wrapper.find(Avatar).simulate('click');
+    act(() => {
```

Comment:
Can you explain again why `enzyme`'s `simulate` function isn't working here anymore? This test is a lot more complex obviously

Reply:
I think this can be simplified now, it was because .simulate('click') gives an error on MUI components - presumably due to MUI longer proving createMount/shallow so I believe I just need to find the html component within it and simulate it on that but I only discovered that later after I had fixed this test. I still have a screenshot of the error: 
![image](https://user-images.githubusercontent.com/90245114/157428156-3c9cd51b-41b0-4494-a6ac-d08e97f11a9e.png)